docs/release-notes/1.5.0-preview2/release-1.5.0-preview2.md

```diff
@@ -0,0 +1,62 @@
+# [ML.NET](http://dot.net/ml) 1.5.0-preview2
+
+## **New Features (IN-PREVIEW, please provide feedback)**
+- **TimeSeriesImputer** ([#4623](https://github.com/dotnet/machinelearning/pull/4623))
```

Comment:
>TimeSeriesImputer [](start = 4, length = 17)

Describe what this transformer is and where it is used and then provide links to samples.